app/views/ReportView/index.jsx

```diff
@@ -213,6 +214,12 @@ const ReportView = () => {
                   )}
                   path={`${path}/immune`}
                 />
+                <Route
+                  render={(routeProps) => (
+                    <Pharmacogenomic {...routeProps} print={false} />
+                  )}
+                  path={`${path}/pharmacogenomic`}
+                />
```

Comment:
non blocker, it doesn't look like `routeProps` is being used in Pharmacogenomic here (it's just to conform to how it's set up)?
but a new way of using Route without the `render` shenanigans is 

```
<Route path={`${path}/pharmacogenomic`}>
  <Pharmacogenomic print={false} />
</Route>
```

and then use hooks inside the component to get the route params you want so it's more clear in the actual component what you need from the router